Civi/Api4/Service/Spec/Provider/ContactGetSpecProvider.php

```diff
@@ -96,7 +98,7 @@ public function modifySpec(RequestSpec $spec) {
     foreach ($entities as $entity => $types) {
       foreach ($types as $type => $info) {
         $name = strtolower($entity) . '_' . $type;
-        $field = new FieldSpec($name, 'Contact', 'String');
+        $field = new FieldSpec($name, 'Contact', 'Integer');
```

Comment:
For the list of affected fields in `$entities`, it does look like `Integer` makes more sense.

And the tests pass...